### ISSUE TITLE
Contributer - Navigation Bar - Navigating Homepage

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/incentivize#49]
## Background 
The Navigation bar for Incentivize.

## Scenario: A logged out user navigates to incentivize.io
- [x] When I am directed to incentivize.io
- [x] Then I see the following links on the Nav 
  - [x] ~About~
  - [x] Discover
  - [x] ~Contact~
  - [x] Connect a Repository
  - [x] Login with <GitHub Icon>

## Scenario: User clicks on Nav Links
- [x] ~When I click About~
- [x] ~Then I am dropped to the About section on the Homepage~

- [x] When I click Discover 
- [x] Then I am taken to the project index page

- [ ] ~When I click Contact~ #104
- [ ] ~Then I am dropped to the Contact Us section of the Homepage~ #104

- [x] When I click Login
- [x] Then I go through the GitHub Auth process and I'm directed back to the Homepage

## Scenario: A logged in user navigates to incentivize.io
- [x] When I am directed to incentivize.io
- [x] Then I see the following links on the Nav 
  - [x] ~About~
  - [x] Discover 
  - [x] ~Contact~
  - [ ] Connect a Repostory
  - [x] My Account

- [x] When I click on My Account
- [x] Then I am directed to my account page

**Comment by brittanygay** [comment]
Pulling this in to work on now

**Comment by brittanygay** [comment]
@Mcscott142 shouldn't we have a "Log Out" link for users who are logged in?

**Comment by Mcscott142** [comment]
@brittanygay Yes we should, I think it's already there but yes, leave it in.

**Comment by brittanygay** [comment]
@Mcscott142 also, we currently don't have a "Contact Us" section on the homepage. Should we be adding this?

**Comment by Mcscott142** [comment]
Yes, we should have a small contact us section with support email being `incentivize@revelry.co`. I can separate into a new ticket if that makes it easier for you.

**Comment by Mcscott142** [comment]
We've got 2 logout buttons. Moving back. 

![incentivize](https://user-images.githubusercontent.com/6877299/43806343-2be9183a-9a69-11e8-9dd1-bd05289ff942.png)

**Comment by brittanygay** [comment]
@Mcscott142 I actually just put in a quick fix for this but it hasn't gotten merged yet. Once my edit is merged and reflected on staging I'll pass back to you.

**Comment by brittanygay** [comment]
Style fix is reflected on staging, this is now ready for QA.

**Comment by gerardramos** [comment]
If I click on Discover and then click on About nothing happens. 

Due to time constraints, please update ac / remove about link from nav and remove the about section from the home page. 

All we need in the nav for a signed out user is Discover and Log In. The logo takes the user back home.

**Comment by Mcscott142** [comment]
Updated #104 to take care of all that. Closing this one.
